tpot/tpot.py

```diff
@@ -288,6 +289,182 @@ def score(self, training_features, training_classes, testing_features, testing_c
 
         return self._evaluate_individual(self.optimized_pipeline_, training_testing_data)[0]
 
+    def export(self, output_file_name):
+        """Exports the current optimized pipeline as Python code.
+
+        Parameters
+        ----------
+        output_file_name: string
+            String containing the path and file name of the desired output file
+
+        Returns
+        -------
+        None
+        """
+        if self.optimized_pipeline_ == None:
```

Comment:
Oh I see;  I thought this way it may be more robust, accounting for empty objects like "", [], {}. But maybe being explicit is not a bad idea here.

 However

    if self.optimized_pipeline_ is None:

is definitely preferred over 

    if self.optimized_pipeline_ == None:

It's more efficient (because you don't have the `__eq__` overhead) and also cleaner style.